src/rest_api.py

```diff
@@ -132,6 +144,9 @@ def _stack_aggregator(handler):
         except KeyError:
             pass
 
+    elapsed_secs = time.time() - stack_aggregator_started_at
+    current_app.logger.info('%s took %0.2f seconds for _stack_aggregators', eri, elapsed_secs)
```

Comment:
Add the elapsed time measurement with in logger statement instead of using separate variable name
```suggestion
    current_app.logger.info('%s took %0.2f seconds for _stack_aggregators', eri, time.time() - stack_aggregator_started_at)
```